communications/shortlinks.csv

```diff
@@ -0,0 +1,12 @@
+keyword,destination_url
+shortlinks,https://github.com/tomeshnet/toronto-community-network/blob/master/communications/shortlinks.csv
+cloud,https://cloud.tomesh.net
+github,https://github.com/tomeshnet
+gh,https://link.tomesh.net/tomeshnet
+chat,https://chat.tomesh.net/#/room/#tomesh:tomesh.net
+tasks,https://github.com/tomeshnet/toronto-community-network/projects/1
+new-task,https://github.com/tomeshnet/toronto-community-network/issues/new?template=task_template.md
+teams,https://github.com/orgs/tomeshnet/teams
+passwords,https://pass.tomeshnet.coop
+calls,https://bbb.hypha.coop/b/tom-v6m-hyj
+general-calls,https://bbb.hypha.coop/b/tom-ygc-ttm
```

Comment:
```suggestion
meeting,https://bbb.hypha.coop/b/tom-v6m-hyj
call,https://bbb.hypha.coop/b/tom-ygc-ttm
```

Reply:
Can i counter like this

- calls - they are not necessary meetings, its a general purpose room. But additional bunch of us will have to remember multiple different short codes.  
- meetings - for official tomesh general meetings?

```suggestion
calls,https://bbb.hypha.coop/b/tom-v6m-hyj
meetings,https://bbb.hypha.coop/b/tom-ygc-ttm
```

Reply:
We can also name rooms. Like private-1 public-1, or physical places like subway stop names like blood dundas. calls and meetings is more confusing as they mean the same thing. In offices we'd give names to meeting rooms like, we can do the same here.

Reply:
I think naming rooms is a good idea, although I'd prefer if the names were more directly applicable, like "comms" or "general", etc.

Reply:
Rooms don't always map to WGs, that's why it's hard to associate them. In reality we only need 2-3 rooms, but we alrdy have 4 WGs and a general meeting. I think it's better to use completely unassociated names. We can stick to our tradition of subway stop names :P
![tomesh0](https://user-images.githubusercontent.com/2261308/91774836-03fcdf80-ebea-11ea-9fa2-c2c5eff8b56f.jpg)

Reply:
the point of these links to make it easier to find rooms.

Last meetup a bunch of us where in the wrong room and thats with only two rooms